modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/fds_view/Filters.tsx

```diff
@@ -25,9 +28,23 @@ import {FDSViewType} from '../FDSViews';
 import {IPickList, getAllPicklists, getFields} from '../api';
 import CheckboxMultiSelect from '../components/CheckboxMultiSelect';
 import OrderableTable from '../components/OrderableTable';
+import ValidationFeedback from '../components/ValidationFeedback';
+
+import '../../css/Filters.scss';
+import '../../css/FDSEntries.scss';
+
+enum filterTypes {
+	DATE_RANGE = 'DATE_RANGE',
+	SELECTION = 'SELECTION',
+}
+
+enum fieldFormats {
+	DATE_TIME = 'date-time',
+	STRING = 'string',
+}
```

Comment:
Enums are nice! :+1:  

We are not 100% consistent with naming in DXP, but most follow `PascalCase`. It might make sense to promote `E` prefix, to be consistent with interfaces and types, but I don't have a strong opinion on this. My personal preference it to have it in singular, as it looks nicer inline in code. All together, these would then be `EFilterType` and `EFieldFormat`.

Reply:
It sounds good, changed!